elasticgraph-indexer_autoscaler_lambda/spec/unit/elastic_graph/indexer_autoscaler_lambda/concurrency_scaler_spec.rb

```diff
@@ -125,13 +141,30 @@ class IndexerAutoscalerLambda
           expect(updated_concurrency_requested_from(lambda_client)).to eq [460] # 500 - 8% since 81/75 = 1.08
         end
 
+        it "resets the concurrency when free storage space drops below the minimum regardless of cpu" do
```

Comment:
```suggestion
        it "pauses indexing when free storage space drops below the minimum regardless of cpu" do
```

"resets the concurrency" is a bit ambiguous--it could be resetting it to some originally high value!